README.md

```diff
@@ -16,26 +16,63 @@ cyber risk reduction.
 Buildroot provides a build target named "legal-info" which produces a set of files including manifest.csv. The legal-info
 is a general "make" target for a Buildroot project.
 
+## Requirements
+
+* `python >= 3.8`
+
+## Installation
+
+Install this from [Python Package Index (PyPI)][link_pypi] using your preferred Python package manager.
+
+install via one of commands:
+
+```shell
+python -m pip install cyclonedx-buildroot   # install via pip
```

Comment:
the registered package might be `CycloneDX-Buildroot`,
but the normalized name is `cyclonedx-buildroot` - so lets use it form maximum compatibility.

see https://packaging.python.org/en/latest/specifications/name-normalization/

Reply:
good